CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
 # This is the main CMake file for NCEPLIBS-ip.
-
+#
 # Mark Potts, Kyle Gerheiser
 cmake_minimum_required(VERSION 3.15)
 
+# Get the version from the VERSION file.
 file(STRINGS "VERSION" pVersion)
 
 project(
   ip
   VERSION ${pVersion}
   LANGUAGES Fortran)
 
+# Set the version for the documentation.
+SET(PACKAGE_VERSION ${pVersion})
```

Comment:
CMake defines `PROJECT_VERSION` and can be used in the Doxyfile.

Reply:
The CMakeLists.txt file contains:

```
# Get the version from the VERSION file.
file(STRINGS "VERSION" pVersion)

project(
  ip
  VERSION ${pVersion}
  LANGUAGES Fortran)

# Set the version for the documentation.
SET(PACKAGE_VERSION ${pVersion})

```

Where is PROJECT_VERSION set? Is it set from the VERSION file? If so, how come we don't use it elsewhere in CMake? We seem to use pVersion here instead.

Reply:
We define the VERSION in a text file named `VERSION`.
This is done so that, if the source code changes, one can bump the VERSION number in this file, without touching the CMakeLists.txt as well as not hard-wiring a version number in that file.
Any of those would work.

CMakeLists.txt reads that file on L7 and sets the variable `pVersion`.
The cmake function `project` sets the `PROJECT` specific variables such as `PROJECT_NAME`, `PROJECT_VERSION`, `PROJECT_MAJOR_VERSION`, etc.

Reply:
OK, done.